kratos/tests/test_obj_io.py

```diff
@@ -226,6 +226,12 @@ def test_ReadObjIOTrianglesDegenerated(self):
         obj_name = GetFilePath("auxiliar_files_for_python_unittest/obj_files/cube_degenerated.obj")
         ReadModelPartFromOBJ(self.model_part, obj_name, True)
 
+        clean_settings = KratosMultiphysics.Parameters("""{
+            "model_part_name"    : "Main",
+            "entity_type"        : "condition"
+        }""")
+        KratosMultiphysics.CleanUpProblematicTrianglesModeler(self.current_model, clean_settings).SetupModelPart()
```

Comment:
I think that we should check this in a separate `test_clean_up_problematic_triangles_modeler.py`. First to be consistent, and secondly to ease an eventual error backtracking.

Reply:
Done